bundler/spec/commands/newgem_spec.rb

```diff
@@ -173,6 +173,18 @@ def gem_skeleton_assertions
     it "generates a default .rubocop.yml" do
       expect(bundled_app("#{gem_name}/.rubocop.yml")).to exist
     end
+
+    it "run rubocop inside the generated gem with no offenses" do
+      prepare_gemspec(bundled_app("#{gem_name}", "#{gem_name}.gemspec"))
+
+      gems = ["rubocop"]
+      path = Bundler.feature_flag.default_install_uses_path? ? local_gem_path(:base => bundled_app("#{gem_name}")) : system_gem_path
+      realworld_system_gems gems, :path => path
+      bundle "install", :dir => bundled_app("#{gem_name}")
+      bundle "exec rubocop", :dir => bundled_app("#{gem_name}")
+
+      expect($?.exitstatus).to eq(0) if exitstatus
+    end
```

Comment:
Right now, this doesn't work. It needs more work & debugging.
I am not entirely sure how to pull this off, so pushed this WIP commit :sweat_smile: 

@simi, any ideas or suggestions?

Reply:
Ahh, I had different idea, to actually create just CI script to install bundler from git and run few commands.

Anyway I'll take a look to this, since ti is probably better.

Reply:
I created #3742, which should fix at least part of the problems that you are having with this spec (currently `realworld_system_gems` does not support a `:path` option).

Reply:
Thanks for this, @deivid-rodriguez! :heart: 
I got the test working :rocket: